web/components/input/buttons/PrimaryButton/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PrimaryButton = ({ text, onClick, disabled }) => (
+  <button
+    onClick={onClick}
+    type="button"
+    className={`primary ${disabled ? 'disabled' : ''}`}>
```

Comment:
Can we disable buttons by doing `<button disabled>Button</button>` instead?
https://stackoverflow.com/questions/14750078/style-disabled-button-with-css

Reply:
That's much cleaner thanks, addressed in 7b2f778